src/EventStore.Core/Services/PersistentSubscription/PersistentSubscriptionService.cs

```diff
@@ -253,11 +249,16 @@ string user
 				ToMessageTimeout(messageTimeoutMilliseconds)
 			);
 
+			if (!result) {
+				onExists($"Group '{groupName}' already exists.");
+				return;
+			}
+
 			Log.Debug("New persistent subscription {subscriptionKey}", key);
 			_config.Updated = DateTime.Now;
```

Comment:
similar to: https://github.com/EventStore/EventStore/pull/3533/files#r988645440
we can refactor this part and call `UpdateSubscriptionConfig(user, stream, groupName, createEntry);`